bindings/python/iota_sdk/types/transaction_options.py

```diff
@@ -46,6 +46,8 @@ def as_dict(self):
         return dict({"strategy": self.name, "value": self.value[0]})
```

Comment:
_to_dict_custom as well

Reply:
good  catch, Actutally this should be `to_dict`, because its not decorated by `dataclass`